RealmSwift/Query.swift

```diff
@@ -289,6 +289,23 @@ extension Query where T == Bool {
     }
 }
 
+// MARK: Mixed
+
+extension Query where T == AnyRealmValue {
+    /// :nodoc:
+    public subscript(position: Int) -> Query<AnyRealmValue> {
+        .init(appendKeyPath("[\(position)]", options: [.isPath]))
+    }
+    /// :nodoc:
+    public subscript(key: String) -> Query<AnyRealmValue> {
+        .init(appendKeyPath("['\(key)']", options: [.isPath]))
+    }
+    /// Query all indexes or keys in a mixed nested collecttion.
+    public var all: Query<AnyRealmValue> {
```

Comment:
This function is `any`, not `all`. `all` would query for all of the values in the dictionary matching the predicate.